src/test/java/junit/edu/umich/its/cpmtest/MigrationTaskServiceTest.java

```diff
@@ -25,14 +39,41 @@ public void tearDown() throws Exception {
 
 	@Test
 	public void test_ExtractEmailSimple() {
-		String email_addr = MigrationTaskService.extractEmailFromToHeader("To: HOWDY@HARVARD.edu");
+		String email_addr = MigrationTaskService.extractEmailFromToHeader("To: HOWDY@ctdev.umich.edu");
 		assertEquals("simple email","HOWDY",email_addr);
 	}
+	@Test
+	public void test_ExtractEmailWithMultipleToAddress() {
+		String email_addr = MigrationTaskService.extractEmailFromToHeader
+				("To: \"Sara M. Mantila\" <msara@umich.edu>, steg_gradstudent <steg_gradstudent@ctools.umich.edu>,ctqa-mbox@ctqa.umich.edu");
+		assertEquals("multiple email","steg_gradstudent",email_addr);
+	}
 
 	@Test
 	public void test_ExtractEmailWithGreaterThan() {
-		String email_addr = MigrationTaskService.extractEmailFromToHeader("To: <HOWDY@HARVARD.edu");
-		assertEquals("email with leading <","HOWDY",email_addr);
+		String email_addr = MigrationTaskService.extractEmailFromToHeader("To: <HOWDY@ctqa.umich.edu>");
+		assertEquals("email with < >","HOWDY",email_addr);
+	}
+
+	@Test
+	public void test_ExtractEmail2() {
```

Comment:
It's dull to do but if the test method names actually describe what is in the test the tests are easier to understand.  If it is hard to come up with a a good name then maybe the method under test does too much.